aspnet/fundamentals/configuration.rst

```diff
@@ -49,36 +48,36 @@ The settings required by your application and the mechanism used to specify thos
 .. note:: You could store your ``Configuration`` instance as a service, but this would unnecessarily couple your application to a single configuration system and specific configuration keys. Instead, you can use the :ref:`Options pattern <options-config-objects>` to avoid these issues.
 
 Using the built-in sources
-----------------------------
+--------------------------
 
-The configuration framework has built-in support for JSON, XML, and INI configuration files, as well as support for in-memory configuration (directly setting values in code) and the ability to pull configuration from environment variables and command line parameters. Developers are not limited to using a single configuration provider. In fact several may be set up together such that a default configuration is overridden by settings from another provider if they are present.
+The configuration framework has built-in support for JSON, XML, and INI configuration files, as well as support for in-memory configuration (directly setting values in code) and the ability to pull configuration from environment variables and command line parameters. Developers are not limited to using a single configuration source. In fact several may be set up together such that a default configuration is overridden by settings from another source if they are present.
 
```

Comment:
I am ok with using the word source in this case. The way I think about it is that the files themselves, the environment or the command line are sources. Providers are the piece of software that allow adapting the data from a source to participate of the configuration system.